llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -935,8 +935,18 @@ void VPlan::prepareToExecute(Value *TripCountV, Value *VectorTripCountV,
   IRBuilder<> Builder(State.CFG.PrevBB->getTerminator());
   // FIXME: Model VF * UF computation completely in VPlan.
   assert(VFxUF.getNumUsers() && "VFxUF expected to always have users");
-  VFxUF.setUnderlyingValue(
-      createStepForVF(Builder, TripCountV->getType(), State.VF, State.UF));
+  Type *TCTy = TripCountV->getType();
```

Comment:
nit: can be defined earlier to also serve TCMO above.

Reply:
Done, thanks!